static/docs/command-reference/repro.md

```diff
@@ -39,6 +39,8 @@ By default, this command recursively searches in pipeline stages, starting from
 the `targets`, to determine which ones have changed. Then it executes the
 corresponding commands.
 
+⚠️ DVC deletes <abbr>output files</abbr> before executing commands.
+
```

Comment:
> It sounds and looks a bit too dramatic. I think it deletes only cached files. Sometimes it means just removing links to cache. If there is a possibility of a data loss it will probably prompt (which might change with DVC deprecating prompts soon and will be failing in that case).

@shcheklein OK so I guess I'll ask maybe @efiop to confirm the behavior? Meanwhile, I've softened the note as you describe above in 99a59b0a.

> Also, it's probably better to say deletes outputs since they can be directories.

Good point! I did a quick review of the "output file" term throughout the docs in e74a207e.